cmd/cli/kubectl-kyverno/commands/apply/command_test.go

```diff
@@ -723,7 +783,66 @@ func Test_Apply_ImageVerificationPolicies(t *testing.T) {
 				},
 			}},
 		},
+		{
+			config: ApplyCommandConfig{
+				PolicyPaths:   []string{"../../../../../test/cli/test-image-validating-policy/empty-message/policy.yaml"},
+				ResourcePaths: []string{"../../../../../test/cli/test-image-validating-policy/empty-message/bad-pod.yaml"},
+				PolicyReport:  true,
+			},
+			expectedReports: []openreportsv1alpha1.Report{{
+				Summary: openreportsv1alpha1.ReportSummary{
+					Pass:  0,
+					Fail:  1,
+					Skip:  0,
+					Error: 0,
+					Warn:  0,
+				},
+			}},
+		},
+		{
+			config: ApplyCommandConfig{
+				PolicyPaths:   []string{"../../../../../test/cli/test-image-validating-policy/empty-message/policy.yaml"},
+				ResourcePaths: []string{"../../../../../test/cli/test-image-validating-policy/empty-message/good-pod.yaml"},
+				PolicyReport:  true,
+			},
+			expectedReports: []openreportsv1alpha1.Report{{
+				Summary: openreportsv1alpha1.ReportSummary{
+					Pass:  1,
+					Fail:  0,
+					Skip:  0,
+					Error: 0,
+					Warn:  0,
+				},
+			}},
+		},
 	}
+	// Test that ImageValidatingPolicy show proper default messages
+	t.Run("ImageValidatingPolicy with empty message", func(t *testing.T) {
+		config := ApplyCommandConfig{
+			PolicyPaths:   []string{"../../../../../test/cli/test-image-validating-policy/empty-message/policy.yaml"},
+			ResourcePaths: []string{"../../../../../test/cli/test-image-validating-policy/empty-message/bad-pod.yaml"},
+			PolicyReport:  true,
+		}
+
+		_, _, _, responses, err := config.applyCommandHelper(os.Stdout)
+		assert.NoError(t, err)
+
+		// Check the responses for the correct message
+		found := false
+		var actualMessage string
+		for _, response := range responses {
+			for _, rule := range response.PolicyResponse.Rules {
+				if rule.Status() == engineapi.RuleStatusFail {
+					found = true
+					actualMessage = rule.Message()
+					assert.Contains(t, actualMessage, "CEL expression validation failed at index",
+						"ImageValidatingPolicy should show default message when message field is empty")
+					assert.NotEmpty(t, actualMessage, "Message should not be empty")
+				}
+			}
+		}
+		assert.True(t, found, "Should have at least one failed rule")
+	})
```

Comment:
I think we should extract the empty message test to its own function, as it validates default error messages, not summary correctness. For example:
```
func Test_ValidatingPolicy_DefaultMessage(t *testing.T) {
	config := ApplyCommandConfig{
		PolicyPaths:   []string{"../../../../../test/cli/test-validating-policy/empty-message/policy.yaml"},
		ResourcePaths: []string{"../../../../../test/cli/test-validating-policy/empty-message/pod-fail.yaml"},
		PolicyReport:  true,
	}

	_, _, _, responses, err := config.applyCommandHelper(os.Stdout)
	assert.NoError(t, err)
       // the rest of the code
       ...
```